doc/contributing/submitting-changes.chapter.md

```diff
@@ -82,6 +82,52 @@ If a security fix applies to both master and a stable release then, similar to r
 
 Critical security fixes may by-pass the staging branches and be delivered directly to release branches such as `master` and `release-*`.
 
+## Deprecating/removing packages {#submitting-changes-deprecating-packages}
+
+There is currently no policy when to remove a package.
+
+It should always be attempted to find a new maintainer or fix smaller issues first.
```

Comment:
Before removing a package, one should try to find a new maintainer or fix smaller issues first.